docs/rules/object-curly-newline.md

```diff
@@ -19,6 +19,7 @@ Or an object option:
 
 * `"multiline": true` (default) requires line breaks if there are line breaks inside properties or between properties
 * `"minProperties"` requires line breaks if the number of properties is more than the given integer
+* `"consistent"` requires that either both curly braces, or neither, directly enclose newlines
```

Comment:
Yes, that's what I want - (in my config) it should never be an error to break properties onto multiple lines, provided there's one per line - it should always be an error to put more than one property on the same line unless the entire object literal is on that line - however, it should also be an error to put too many properties in a one-line object literal.